manifest/resolver.go

```diff
@@ -50,6 +50,23 @@ type Config struct {
 	Arch string
 }
 
+// Platform describes a system where a package can be installed
+type Platform struct {
+	// OS is the operating system of the platform
+	OS string
+	// Arch is the CPU architecture of the platform
+	Arch string
+}
+
+// HermitPlatforms are core platforms officially supported by Hermit.
+// For a package to be considered fully compliant, these platforms need to be supported
+var HermitPlatforms = []Platform{
+	{"linux", "x86_64"},
+	{"linux", "aarch64"},
```

Comment:
Can we get rid of linux/arm64 for now? I think darwin/arm64 is reasonable but linux/arm64 seems much more niche and will introduce quite a lot of warnings.